cg/store/filters/status_order_filters.py

```diff
@@ -28,6 +28,20 @@ def filter_orders_by_ticket_id(orders: Query, ticket_id: int | None, **kwargs) -
     return orders.filter(Order.ticket_id == ticket_id) if ticket_id else orders
 
 
+def filter_orders_by_search(orders: Query, search: str | None, **kwargs) -> Query:
+    return (
+        orders.filter(
+            or_(
+                Order.id.startswith(search),
+                Order.customer.name.ilike(f"%{search}%"),
+                Order.ticket_id.ilike(f"%{search}%"),
```

Comment:
I'm not 100 % sure how we want to do this (in terms of which operators to use,`begin`/`ilike` or something else)

Reply:
Personally I prefer `.contains()`. It feels the most pythonic and I don't think we need anything more advanced. Maybe case insensitivity is good though. Also `ilike` was giving me some issues a while ago

Reply:
I don't really see the use in not setting == all around but on the other hand someone from production is probably better suited to answer this

Reply:
ooh, I found the `icontains()` function which does what you expect 😁

Reply:
> Personally I prefer `.contains()`. It feels the most pythonic and I don't think we need anything more advanced. Maybe case insensitivity is good though. Also `ilike` was giving me some issues a while ago

So you don't like I like? 🤪

Reply:
Settled on exact matching for order id and ticket id. Using `icontains` for the customer internal id.
Will check with production if they want to adjust the search.